app-text/asciidoc/asciidoc-9.0.0_rc1.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} pypy3 )
+
+inherit autotools python-single-r1 readme.gentoo-r1
+
+DESCRIPTION="A plain text human readable/writable document format"
+HOMEPAGE="http://asciidoc.org/ https://github.com/asciidoc/asciidoc-py3/"
+SRC_URI="https://github.com/${PN}/${PN}-py3/archive/9.0.0rc1.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sh ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="examples graphviz highlight test"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	app-text/docbook-xml-dtd:4.5
+	>=app-text/docbook-xsl-stylesheets-1.75
+	dev-libs/libxslt
+	dev-libs/libxml2
+	${PYTHON_DEPS}
+	graphviz? ( media-gfx/graphviz )
```

Comment:
Should probably be an `optfeature` since it doesn't have any function in this ebuild, 
https://projects.gentoo.org/qa/policy-guide/dependencies.html#optional-runtime-dependencies

Reply:
Yeah, I could change that, but it's always been like that in asciidoc.  But I guess a version bump is as good an opportunity as any.  I'll do it in a clean-up commit though, that will consolidate some of your other suggestions (the ones that aren't directly relevant to the version bump itself).

Reply:
One question: will removing the USE flag cause a problem for the asciidoc `package.use.mask` entry in `profiles/arch/riscv/package.use.mask`?

Reply:
It will be fine, there's still versions left in tree. But I wasn't aware of such mask, it's good to know so it may be removed once obsolete.

Reply:
OK